awx/main/models/inventory.py

```diff
@@ -6,7 +6,6 @@
 import time
 import logging
 import re
-import copy
```

Comment:
This is not related to the PR. Let's keep it and handle the unused imports separately.

Reply:
Thank you! This should be resolved via this commit https://github.com/ansible/awx/pull/15537/commits/dbba124bf8eb6783df25a71849cec738c91cda1f

Reply:
ack, let's resolve